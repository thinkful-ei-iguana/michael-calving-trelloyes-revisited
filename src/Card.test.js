//smoke test and snapshot test
import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

it('renders without crashing', () =>{
  const div = document.createElement('div');
  ReactDOM.render(<Card />, div);
  ReactDOM.unmountComponentAtNode(div);
});


it('renders the UI as expected', () => {
  const tree = renderer.create(<Card name="Card" title="card 1" content="some content"/>).toJSON();
  expect(tree).toMatchSnapshot();  
  });
