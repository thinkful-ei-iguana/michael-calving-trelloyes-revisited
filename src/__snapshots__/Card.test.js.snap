// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    card 1
  </h3>
  <p>
    some content
  </p>
</div>
`;
